### PR TITLE
Feat: Add tests and functionality for user story 9

### PR DESCRIPTION
**Pull request by MatthewTLim**
# Describe the changes below:
This PR will add all tests and functionality for user story 9
# Relevant user story:
User Story 9
# Specific feedback requests? Include file name and line:

# Before submitting, check the following:
- [X] Entire test suite is passing
- [99.57] SimpleCov test percentage

# Note any co-authors / anything else you'd like to add:

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -16,9 +16,14 @@ def edit
   def update
     @merchant = Merchant.find(params[:merchant_id])
     @item = Item.find(params[:id])
+    # require 'pry'; binding.pry
```
Comment (awiley33) [inline app/controllers/items_controller.rb]: oops there's a pry left in here!

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -27,7 +32,7 @@ def update
   private
 
   def item_params
-    params.require(:item).permit(:name, :description, :unit_price, :id, :merchant_id)
+    params.require(:item).permit(:name, :description, :unit_price, :id, :merchant_id, :status)
```
Comment (awiley33) [inline app/controllers/items_controller.rb]: nice addition of status to strong params

File: app/views/items/index.html.erb [hunk 3]
```diff
@@ -3,6 +3,11 @@
 <h2>List of Items</h2>
 <% @merchant.items.each do |item| %>
   <%= link_to item.name, merchant_item_path(@merchant, item) %>
+  <% if item.status == "disabled" %>
+    <%= button_to "Enable #{item.name}", merchant_item_path(@merchant, item), method: :patch, params: { item: { status: "enabled" } }  %>
+  <% else %>
+    <%= button_to "Disable #{item.name}", merchant_item_path(@merchant, item), method: :patch, params: { item: { status: "disabled" } }  %>
+  <% end %>
```
Comment (awiley33) [inline app/views/items/index.html.erb:12]: good job figuring this out!!

**Review by awiley33 — APPROVED** [review]
Nice Work!

File: spec/features/merchant_spec.rb [hunk 4]
```diff
@@ -89,4 +90,34 @@
...
+              click_button("Enable #{@item_1.name}")
+
+              @item_1.reload
+
+              expect(current_path).to eq("/merchants/#{@merchant_1.id}/items")
+              expect(page).to have_button("Disable #{@item_1.name}")
+              expect(page).to have_button("Enable #{@item_2.name}")
+              expect(page).to have_button("Enable #{@item_3.name}")
+            end
+          end
```
Comment (westonio) [inline spec/features/merchant_spec.rb]: Nice work here! 

If you get a chance to refactor later, I would recommend potentially breaking these into smaller tests where possible.

**Review by westonio — APPROVED** [review]

**Pull request merged**